packages/ember-metal/lib/meta.js

```diff
@@ -33,7 +32,7 @@ if (DEBUG) {
 @module ember-metal
 */
 
-export const UNDEFINED = symbol('undefined');
```

Comment:
I assumed that, checking equality by reference is faster than checking equality between strings for example, it maybe consumes less memory because strings are not copied around